apps/api/openapi.yaml

```
@@ -2759,6 +2760,7 @@ paths:
           description: Invalid ID supplied
         '404':
           description: Game not found
+       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)